### PR TITLE
Add new option --page-version to set the PAGE version

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by stweil** [comment]
(Most) PAGE files from Transkribus can be handled like this:

    python page2img.py --page-version=2013-07-15 ONB_ibn_18640702_011.xml

**Comment by stweil** [comment]
I added a 2nd commit which fixes the processing for most remaining Transkribus PAGE files (for example ONB_ibn_19110701_001.xml). Those files have an empty Unicode field like in the following data:

         <TextLine id="tl_233"
                   primaryLanguage="German"
                   custom="readingOrder {index:0;}">
            <Coords points="1024,4607 1050,4607 1050,4637 1024,4637"/>
            <Baseline points="1024,4637 1050,4637"/>
            <TextEquiv>
               <Unicode/>
            </TextEquiv>
            <TextStyle fontFamily="Times New Roman"
                       fontSize="7.5"
                       bold="true"
                       italic="true"/>
         </TextLine>

File: page2img.py [hunk 1]
```diff
@@ -114,7 +116,7 @@ def cli(page, out_dir, level, image_format, text, font):
         # extract text if requested by user
         if text:
             unic = struct.find("./" + PC + "TextEquiv").find("./" + PC + "Unicode")
-            if unic is not None:
+            if unic is not None and unic.text is not None:
```
Comment (stweil) [inline page2img.py:119]: Would it be better to `print` a warning if an empty text was found?

**Comment by stweil** [comment]
There still remains an exception caused by empty Coords points in like in the following data:

         <TextLine id="line_1548641918006_392"
                   primaryLanguage="German"
                   custom="readingOrder {index:152;}">
            <Coords points=""/>
            <Baseline points="869,5084 929,5083 982,5082 1251,5078 1260,5077"/>
            <TextEquiv>
               <Unicode>detto inh. 50 J. v. K. 4</Unicode>
            </TextEquiv>
            <TextStyle fontFamily="Times New Roman" fontSize="4.5"/>
         </TextLine>

File: page2img.py (hunk 1 above)
Comment (wrznr) [reply page2img.py:119]: No. It's better like this.

**Review by wrznr — APPROVED** [review]
Great work!

**Pull request merged**